catalog/imaging/imaging-plugin-nitf/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -14,17 +14,29 @@
  -->
 <metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
 
-    <OCD name="Nitf Imaging Input Transformer"
-         id="NITF_Input_Transformer">
+    <OCD name="Nitf Render Plugin"
+         id="NITF_Render_Plugin">
         <AD
                 description="Maximum length of longest side of NITF overview image in pixels. The input transformer will calculate the size of the shorter side so that the overview will have the same aspect ratio as the original."
                 name="Overview image maximum side length (pixels)" id="maxSideLength" required="true"
                 type="Integer" default="1024"/>
+        <AD
+                description="Maximum NITF file size in Megabytes to create thumbnails/overviews for. Rendering large nitfs can cause out-of-memory exceptions."
+                name="Maximum NITF file size to process in Megabytes" id="maxNitfSizeMB" required="true"
+                type="Integer" default="10"/>
```

Comment:
The answer to this question isn't going to be a single number. We'll want to have that value for each type of image (image mode, image compression, etc) within a NITF.